File: env_variable/ocaml_env_variable/env_variable.ml
(* comment *)

let env_variable_function(n) = if n < 0 then abs n else (n + n - n) * (n / 1) * (int_of_float 1.25);;

let main () =
  if  Array.length Sys.argv <> 1 then
    begin
      Printf.printf "%s <noargs>\n" Sys.argv.(0);
      exit 1
    end
  else
    begin
    for idx_i = 1 to 9 do
      Printf.printf "env_variable_function(%d)=<%d>\n" idx_i (env_variable_function(idx_i));
      Printf.printf "%s\n" (Sys.getenv "OS")
    done;
    end

let _ = main ()

File: env_variable/golang_env_variable/env_variable.go
package main

//comment
import "os"
import "fmt"
import "math"

const FLOAT_VAL = 1.25

func floor(x float32) float32 {
  return float32(math.Floor(float64(x)))
}

func abs(x int) float32 {
  return float32(math.Abs(float64(x)))
}

func env_variable_function(n int) int {
  x := 0

  x = (n + n - n) * (n / 1)
  x = x * int(floor(FLOAT_VAL))

  if( x < 0 ) {
    x = int(abs(x))
  }
  return x
}

func main() {
  args := os.Args
  argc := len(args)

  if argc != 1 {
    fmt.Println(args[0] + " <noargs>")
    os.Exit(1)
  }

  os.Setenv("FOO", "1")
  fmt.Println("OS=" + os.Getenv("OS"))
}
